Mobilefirst finallayout

pushkar1995 opened the pull request:
- Add the My Recent Works section for the portfolio🚀
- Add an About Me section for the portfolio 💯

Efiamotu-1 commented on style.css:
- [ ] You have not implemented the styling for pressed state as shown in the button interaction requirements below, you can do this by making use of the css `:active` pseudo selector. please ensure you implement this following the provided styling on figma for all buttons in your application :+1: :sparkles: 
- button interaction
![act](https://github.com/pushkar1995/portfolio-website1.0/assets/89055502/4098cb20-8ead-4226-ab5e-dd328fca9757)

Efiamotu-1 reviewed (changes requested):
Hi @pushkar1995 , 

![](https://user-images.githubusercontent.com/89055502/221963929-dc172827-5c2c-411b-bf4d-923659f626e1.gif) 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [![@Efiamotu-1](https://img.shields.io/github/followers/Efiamotu-1?label=Habeeb-Efiamotu-Musa&style=social)](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [![@EFYAMOTU](https://img.shields.io/twitter/follow/omarramoun?label=Habeeb-Efiamotu-Musa&style=social)](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[![Habeeb Efiamotu Musa](https://img.shields.io/badge/linkedin-@Habeeb-crimson)](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [![Habeeb Efiamotu Musa ](https://img.shields.io/badge/slack-@Habeeb-brightgreen)](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

pushkar1995 commented:
@Efiamotu-1  I have changed the font family of all paragraphs to Inter according to the design and added the : active CSS class to all the buttons according to the review asked.

wayungi reviewed (approved):
Hi @pushkar1995 , 

![image](https://user-images.githubusercontent.com/99656687/220082459-c1b049c7-5498-42e7-942b-ec7a05d3e603.png)

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] Profesional README ✔️ 
- [ ] All linter checks are passing ✔️ 
- [ ]Correct font applied ✔️ 
- [ ] No merge conflicts ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
